www/layouts/pages/single.html

```diff
@@ -2,7 +2,7 @@
 <div class="page-single">
   {{ block "header" . }}
   {{ partialCached "header" . }}
-  {{ partialCached "page_title" . }}
+  {{ partialCached "page_title" . .Title}}
```

Comment:
I suspect we're not likely to render the `"page_title"` partial with the same `.Title` more than once - maybe it would be better to just remove `partialCached` entirely?

Reply:
Yeah i dont think the it improves the performance that much and it would also probably costs more in memory then it saves on performance.

Reply:
@Wassaf-Shahzad in the very beginning of this project, we were trying to build all course sites as a single Hugo build so using `partialCached` was much more important.  Nowadays each course build is run individually, so caching partials is less effective.  I think it's okay to remove it, especially where it's causing issues like this.

Reply:
my guess is that for most of our partials the caching is going to give us only very, very marginal build time increases given the overhead of all the stuff that's happening other than the Hugo build itself

@gumaerc this makes me think, back when we were doing the full site build we ended up using `partialCached` to render the course nav which, iirc, prevented us from setting class names for highlighting current nav items in HTML, so we had to lean on some JS instead to do that once the page had loaded in the user's browser. Should we revisit that perhaps? It is nice to have a pure-HTML nav-highlighting thing because then users don't need to use JS in order for the nav to work for them

Reply:
@alicewriteswrongs Yea, that sounds about right.  Nowadays the Hugo build's performance is the least of our concerns.